packages/larva-patterns/modules/carousel-grid/carousel-grid.twig

```diff
@@ -0,0 +1,17 @@
+<div class="carousel-grid //">
+	<div class="lrv-a-wrapper">
+		<div class="lrv-a-carousel-grid {{ carousel_grid_overlay_modifier_class1 }}">
```

Comment:
Usually the {{ modifier_class }} is on the element that has the class name of the module, in this case `{{ carousel_grid_classes }}` and on the outermost element. Inner modifier classes like this are named according to the element in the pattern, like `{{ carousel_grid_primary_classes }}`. This one is containing primary, so maybe something like `{{ carousel_grid_layout_classes }}`. Whatever you think is best within these guides - it's definitely not perfect!